dts/bindings/mipi-dsi/st,stm32-mipi-dsi.yaml

```
@@ -0,0 +1,53 @@
+#
+# Copyright (c) 2023 bytes at work AG
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+description: STM32 MIPI DSI host
+
+compatible: "st,stm32-mipi-dsi"
+
+include: mipi-dsi-host.yaml
+
+properties:
+  clocks:
+    required: true
+
+  hs-active-high:
+    type: boolean
+    description:
```

```suggestion
    description: |
```
(adjust others as well)